### PR TITLE
API Endpoint to Retrieve Church Data

### LINKED ISSUE #48: API Endpoint to Retrieve Church Data
App needs an endpoint to retrieve church name and potential other church related data.

### PR DESCRIPTION
**Pull request by mattash**
Fixes #48

### CODE REVIEW

**Review by andylitalo — CHANGES_REQUESTED** [review]
Just a note about the get_queryset()

**Review by andylitalo — CHANGES_REQUESTED** [review]
Comment about possible redundancy

**Comment by andylitalo** [comment]
Sorry for the review spam--still figuring out GitHub's mobile app

File: hub/views/church.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    Responses:
+        - 200 OK: Successfully retrieved the church's information.
+        - 404 Not Found: Church with the specified ID does not exist.
+    """
+    serializer_class = ChurchSerializer
+    permission_classes = [permissions.AllowAny]
+    queryset = Church.objects.all()
+
+    def get_queryset(self):
```
[the remark this answers is not in the document]
Comment (mattash) [reply hub/views/church.py:56]: So the way this should work is that a request of /api/churches/ will return a list of all churches in the DB. /api/churches/[ID], will return the information a specific church.

File: hub/views/church.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        """
+        Optionally restricts the returned churches to a given user,
+        by filtering against a `username` query parameter in the URL.
+        """
+        queryset = Church.objects.all()
+        return queryset
```
Comment (andylitalo) [inline hub/views/church.py:62]: nit could you reduce to one line `return Church.objects.all()`?

File: hub/views/church.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (andylitalo) [reply hub/views/church.py:56]: Thanks! Could we include that in the docstring? It currently only describes filtering by `username`, which I'm not sure if this method supports.

**Review by andylitalo — CHANGES_REQUESTED** [review]
Thanks for the explanation! A few more questions about getting the queryset

**Review by andylitalo — CHANGES_REQUESTED** [review]
A couple questions

**Review by andylitalo — APPROVED** [review]
Tested church detailed view with condensed queryset param

**Pull request merged**